tests/Install-Scripts.Test/GivenThatIWantToGetTheSdkLinksFromAScript.cs

```diff
@@ -91,6 +94,32 @@ public void WhenRuntimeParametersArePassedToInstallScripts(string runtime, strin
             commandResult.Should().HaveStdOutContainingIgnoreCase(runtimeType);
         }
 
+
+        [Theory]
+        [InlineData("3.1", _feedCredential)]
+        [InlineData("5.0.2xx", _feedCredential)]
+        [InlineData("LTS", "?" + _feedCredential)]
+        [InlineData("release/1.0.0", "?" + _feedCredential)]
```

Comment:
These have extra `?`. Doesn't this break the test?

Reply:
No, it does not. The previous behavior is that credentials should have "?" in the beginning. I added an opportunity to pass credentials without it, but did not change the previous behavior, so credentials still could be starting "?", for the backward compatibility. Now you can pass it either way, with or without "?".